src/components/CommentList.tsx

```diff
@@ -0,0 +1,21 @@
+import React, { FC } from 'react';
+import { Comments } from '../interfaces';
+import { Comment } from './Comment';
+
+interface Props {
+  commentsToPost: Comments[];
+}
+
+export const CommentList: FC<Props> = ({ commentsToPost }) => (
+
+  <>
```

Comment:
you don't need this

Reply:
If I remove react fragment here, my app will crash.
I have tried many times, but I couldn't do it.

Reply:
if you remove React.Fragment, you need to remove curly braces as well